Resources/translations/time.tr.xliff

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>diff.ago.year</source>
+                <target>1 yıl önce|%count% yıl önce</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>diff.ago.month</source>
+                <target>1 ay önce|%count% ay önce</target>
+            </trans-unit>
+            <trans-unit id="3">
+                <source>diff.ago.day</source>
+                <target>1 gün önce|%count% gün önce</target>
+            </trans-unit>
+            <trans-unit id="4">
+                <source>diff.ago.hour</source>
+                <target>1 saat önce|%count% saat önce</target>
+            </trans-unit>
+            <trans-unit id="5">
+                <source>diff.ago.minute</source>
+                <target>1 dakika önce|%count% dakika önce</target>
+            </trans-unit>
+            <trans-unit id="6">
+                <source>diff.ago.second</source>
+                <target>1 saniye önce|%count% saniye önce</target>
+            </trans-unit>
+            <trans-unit id="7">
+                <source>diff.empty</source>
+                <target>şimdi</target>
+            </trans-unit>
+            <trans-unit id="8">
+                <source>diff.in.second</source>
+                <target>yaklaşık 1 saniye|yaklaşık %count% saniye</target>
+            </trans-unit>
+            <trans-unit id="9">
+                <source>diff.in.hour</source>
+                <target>yaklaşık 1 saat|yaklaşık %count% saat</target>
+            </trans-unit>
+            <trans-unit id="10">
+                <source>diff.in.minute</source>
+                <target>yaklaşık 1 dakika|in %count% dakika</target>
```

Comment:
@imanalopher can you tell me if this translation is better than this other translation? https://github.com/KnpLabs/KnpTimeBundle/pull/76/files#diff-892618d3099db22dd8ec160b4a0b70b5

There are some differences - I'd like to know which are the better translations :).

Thanks!